packages/react-native-audio-api/ios/audioapi/ios/system/AudioSessionManager.mm

```diff
@@ -24,9 +24,10 @@ - (instancetype)initPrivate
   if (self = [super init]) {
     self.audioSession = [AVAudioSession sharedInstance];
 
-    self.sessionCategory = AVAudioSessionCategoryPlayback;
+    self.sessionCategory = AVAudioSessionCategoryPlayAndRecord;
     self.sessionMode = AVAudioSessionModeDefault;
-    self.sessionOptions = AVAudioSessionCategoryOptionDuckOthers | AVAudioSessionCategoryOptionAllowBluetooth;
+    self.sessionOptions = AVAudioSessionCategoryOptionAllowAirPlay | AVAudioSessionCategoryOptionAllowBluetoothA2DP |
+        AVAudioSessionCategoryOptionDefaultToSpeaker;
```

Comment:
I think we should stay with 

AVAudioSessionCategoryPlayback
AVAudioSessionModeDefault
AVAudioSessionCategoryOptionAllowAirPlay

as default, and only change that for microphone example

Reply:
Reason:
- we want to stay as close as to default configuration, in our default settings (do not set something lib user doesn't expect)
- libs main purpouse is playback not p&r